src/main/java/seedu/address/model/client/Client.java

```diff
@@ -34,7 +36,7 @@
      * Every field must be present and not null.
      */
     public Client(Name name, Phone phone, Email email, Address address, Country country, Timezone timezone,
-            ContractExpiryDate contractExpiryDate, LastModifiedInstant lastModifiedInstant) {
+                  ContractExpiryDate contractExpiryDate, LastModifiedInstant lastModifiedInstant) {
```

Comment:
Is this 8 spaces after the previous line?

Reply:
resolved